includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -369,18 +372,52 @@ protected function get_row_data( $item ) {
 					$course_title = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . apply_filters( 'the_title', $item->post_title, $item->ID ) . '</a></strong>';
 				}
 
-				$column_data = apply_filters(
-					'sensei_analysis_overview_column_data',
-					array(
-						'title'              => $course_title,
-						'last_activity'      => $last_activity_date,
-						'completions'        => $course_completions,
-						'average_percent'    => $average_grade,
-						'days_to_completion' => $average_completion_days,
-					),
-					$item,
-					$this
+				// Fetch learners in course.
+				$course_args           = array(
+					'post_id' => $item->ID,
+					'type'    => 'sensei_course_status',
+					'status'  => 'any',
 				);
+				$course_students_count = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_learners', $course_args, $item ) );
+
+				// Calculate average lesson progress per student for course.
+				$average_course_progress = 0;
+
+				// Get all course lessons.
+				$lessons        = Sensei()->course->course_lessons( $item->ID, 'any', 'ids' );
```

Comment:
Using status `any` will include draft lessons into the calculation. I'm not sure if this is ok or not.